tests/common.bash

```diff
@@ -22,103 +69,65 @@ function kubectl_apply_should_fail_with_message {
+	kubectl wait --timeout $TIMEOUT --for=condition="$1" clusteradmissionpolicies --all
 }
 
 function wait_for_default_policy_server_rollout {
-	wait_for_policy_server_rollout default
+ 	revision=$(kubectl -n $NAMESPACE get "deployment/policy-server-default" -o json | jq -r '.metadata.annotations."deployment.kubernetes.io/revision"')
+ 	wait_rollout -n $NAMESPACE --revision $revision "deployment/policy-server-default"
 }
 
 function default_policy_server_rollout_should_fail {
-	revision=$(kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE get "deployment/policy-server-default" -o json | jq ".metadata.annotations.\"deployment.kubernetes.io/revision\"" | sed "s/\"//g")
-	run kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE rollout status --revision $revision --timeout $TIMEOUT "deployment/policy-server-default"
-	[ "$status" -ne 0 ]
-}
-
-function wait_for_policy_server_rollout {
-	revision=$(kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE get "deployment/policy-server-$1" -o json | jq ".metadata.annotations.\"deployment.kubernetes.io/revision\"" | sed "s/\"//g")
-	run kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE rollout status --revision $revision "deployment/policy-server-$1"
+	revision=$(kubectl -n $NAMESPACE get "deployment/policy-server-default" -o json | jq -r '.metadata.annotations."deployment.kubernetes.io/revision"')
+	run ! kubectl -n $NAMESPACE rollout status --revision $revision --timeout $TIMEOUT "deployment/policy-server-default"
 }
 
 function default_policy_server_should_have_log_line {
-	policy_server_should_have_log_line default "$1"
-}
-
-function policy_server_should_have_log_line {
-	run kubectl --context $CLUSTER_CONTEXT logs -n $NAMESPACE -lapp="kubewarden-policy-server-$1" | grep "$2"
+	kubectl logs -n $NAMESPACE -lapp="kubewarden-policy-server-default" | grep "$1"
 }
 
 function create_configmap_from_file_with_root_key {
-	run kubectl --context $CLUSTER_CONTEXT -n $NAMESPACE create configmap $1 --from-file=$2=$3
+	kubectl -n $NAMESPACE create configmap $1 --from-file=$2=$3
 }
 
-function kubewarden_crds_should_not_be_installed {
-	run kubectl --context $CLUSTER_CONTEXT get crds policyservers.policies.kubewarden.io clusteradmissionpolicies.policies.kubewarden.io admissionpolicies.policies.kubewarden.io
-}
-
-
 function install_kubewarden_stack {
-	run helm install --kube-context $CLUSTER_CONTEXT --wait -n kubewarden --create-namespace kubewarden-crds kubewarden/kubewarden-crds
-	run helm install --kube-context $CLUSTER_CONTEXT --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
-	run helm install --kube-context $CLUSTER_CONTEXT --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defaults
+	helm install --wait -n kubewarden --create-namespace kubewarden-crds kubewarden/kubewarden-crds
+	helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
+	helm install --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defaults
+	wait_pods
```

Comment:
in theory, the `helm install --wait` should do the waiting already.

Reply:
You are probably right, we also wait before each test case, I will remove this.

Reply:
I put it back :)